templates/install.html

```diff
@@ -97,8 +97,8 @@ <h3 class="p-list-step__title">
                       <li>Select the subnet where to create the DHCP Dynamic range on.</li>
                       <li>Fill in the details for the dynamic range.</li>
                   </ol>
-                  <p><a href="https://assets.ubuntu.com/v1/57bf7d7e-Turn+on+DHCP.png" class="venobox venobox--expand" data-gall="installGallery" title="VLAN details page in an active MAAS">
-                      <img class="p-image--bordered" src="https://assets.ubuntu.com/v1/57bf7d7e-Turn+on+DHCP.png?w=552" class="image--shadow" alt="VLAN details">
+                  <p><a href="{https://assets.ubuntu.com/v1/6727ac9a-2.6-enable-dhcp.png" class="venobox venobox--expand" data-gall="installGallery" title="VLAN details page in an active MAAS">
```

Comment:
Got a rogue curly brace here, other than that LGTM

Reply:
thanks, fixed